masonite/routes.py

```diff
@@ -1,23 +1,26 @@
-''' Module for the Routing System '''
+""" Module for the Routing System """
 import cgi
 import importlib
 import re
 from pydoc import locate
 
 from config import middleware
 
-import cgitb; cgitb.enable()
+import cgitb
+
+cgitb.enable()
```

Comment:
lets get rid of this import. I'm pretty sure it's not needed. as well as the cgitb.enable thing. Just delete them